### PR TITLE
Get Tests Green

### PR DESCRIPTION
**Pull request by seanpdoyle**
* Remove duplicate files
* fix imports

### CODE REVIEW

**Comment by seanpdoyle** [comment]
@stefanpenner I got the suite green, but I'm not sure if this is the correct fix, but :green_apple: :green_book: :green_heart:

File: tests/autorun_test.js [hunk 1]
```diff
@@ -1,25 +0,0 @@
-import Backburner from 'backburner';
```
Comment (stefanpenner) [inline tests/autorun_test.js]: why are these removed?

**Comment by stefanpenner** [comment]
awesome, good enough for now :)
